src/library_html5_webgl.js

```diff
@@ -313,6 +316,7 @@ var LibraryHtml5WebGL = {
   },
 
   emscripten_webgl_destroy_context__proxy: 'sync_on_webgl_context_handle_thread',
+  emscripten_webgl_destroy_context__deps: ['free'],
```

Comment:
The dep here comes from `GC.deleteContext`. Can we not add it to the deps there, which would be more direct? Also, it is only relevant when pthreads are used, I see in the code.

Reply:
Fixed by extracting GL_deleteContext into a separate top level function.